### ISSUE TITLE
Landmarks without aria-label not shown in Voiceover rotor

### ISSUE THREAD

**Issue by jonnywyatt** [alphagov/govuk-frontend#2184]
## Description of the issue
On gov.uk, only the landmarks that have `aria-labelledby` are recognised by Voiceover and listed in the landmarks section of the Rotor.
Landmarks that don't have the attribute eg `<main id="content" role="main />`, are ignored.

## Steps to reproduce the issue
See screen recording here - https://ukgovernmentdigital.slack.com/archives/C0647LW4R/p1618226743059000?thread_ts=1618217541.053300&cid=C0647LW4R 
1. Navigate to gov.uk home page
2. Open Voiceover then the Rotor
3. Switch to Landmarks tab

## Actual vs expected behaviour
Expected - every landmark element with a role should be listed
Actual - only landmarks with `aria-labelledby` set, are listed.

## Environment (where applicable)
- Operating system: Mac OS Catalina 10.15.6
- Browser: Safari
- Browser version: 14.0 (15610.1.28.1.9, 15610), no extensions loaded
- GOV.UK Frontend Version: seen on gov.uk live home page

**Comment by joelanman** [comment]
![screenshot of Safari and Voiceover showing the issue](https://user-images.githubusercontent.com/1132904/114391059-ef74c280-9b8e-11eb-904b-22b7c0d78afa.png)

**Comment by jonnywyatt** [comment]
@36degrees correctly pointed out that this is due to the Verbosity setting in Voiceover - I had it set to low, but if you change it to medium then all landmarks are reported.

**Comment by joelanman** [comment]
@jonnywyatt thanks for reporting, now we have it documented if something like this comes up again
